play-media-native/store/filters.ts

```diff
@@ -26,10 +32,21 @@ export const filtersSlice = createSlice({
     setEventFiltersActive: (state: FiltersState, action: PayloadAction<number>) => {
       state.eventFiltersActive = action.payload;
     },
+    setAthleteFilterValues: (state: FiltersState, action: PayloadAction<IIndexable>) => {
+      state.athleteFilterValues = action.payload;
+    },
+    setEventFilterValues: (state: FiltersState, action: PayloadAction<IIndexable>) => {
+      state.eventFilterValues = action.payload;
+    },
   },
 });
 
-export const { toggleVisible, setAthleteFiltersActive, setEventFiltersActive } =
-  filtersSlice.actions;
+export const {
+  toggleVisible,
+  setAthleteFiltersActive,
```

Comment:
This is not good. It won't scale well for a starter. Simply imagine more entities needing to be filtered (think of adding media filtering to global state). We would have to add setMediaFilteresActive, setMediaFIlterValues to an already verbose API. Make it abstract instead.

Reply:
Not sure what you mean by this. Media filtering would also need to be in the filters store, and it makes sense, as it concerns filtering.

Reply:
filtersActive = [eventListingID, mediaListingID, addEventID]

Instead of always having to go and add new props here.

Also what would any CH ONE client (wanting to reuse some of the demo code) have to do for a completely different data structure in CH ONE (one that does not contain events and athletes)?

Make it less opinionated.

Reply:
That's highly unlikely, clients reusing demo assets. Demo assets are always intended for a single purpose.

Not sure what you mean by this "filtersActive = [eventListingID, mediaListingID, addEventID]". This filters store contains information about the location and sport IDs, or nationality and sport IDs used to filter the data displayed to the user.